aima-core/src/main/java/aima/core/agent/DTAgent.java

```diff
@@ -0,0 +1,62 @@
+package aima.core.agent;
+
+
+import aima.core.agent.Action;
+import aima.core.agent.Percept;
+import aima.core.agent.impl.AbstractAgent;
+import aima.core.probability.BeliefState;
+
+import java.util.List;
+
+/**
+ * Artificial Intelligence A Modern Approach (3rd Edition): Figure 13.1, page 484.<br>
+ * 
+ * Figure 13.1  A decision-theoretic agent that selects rational actions.

```

Comment:
Pseudocode is missing from the javadoc comments.

Reply:
I've added that, thanks @samagra14

Reply:
Looks good.